04/styles/normal.css

```diff
@@ -0,0 +1,36 @@
+@media (min-width: 600px) {
+    .hamburger-icon {
+        display: none;
+    }
+    .navbar {
+        margin: 0 auto;
+    }
+    .navbar__menu {
+        display: flex;
+        margin: 0;
+        background: #e2a258;
+        padding: 10px;
```

Comment:
Przez ten padding pojawił się poziomy scroll, bo 100% width + 10px padding to wiecej niż 100%.
Jeśli chcesz, aby, ten problem nie występował to wystarczy ustawić `box-sizing: border-box`